prisondersdilemma.py

```diff
@@ -11,182 +11,260 @@
 from numpy.random import shuffle
 import numpy as np
 import random
+from win32api import GetSystemMetrics
 
 #these are all just building blocks for now, not functioning code! it won't run yet
 
 #create window
 
```

Comment:
If you want too, you can separate 'specs' into separate .py files by defining them as functions/classes. Then you can import them as you would any other library. May help you keep track if your code gets very long